.github/workflows/wheels.yml

```
@@ -22,7 +22,7 @@ jobs:
     - name: Build wheels
       uses: pypa/cibuildwheel@v2.1.1
       env:
-        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* pp* *-win32 cp310-manylinux_i686"
+        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* cp38-manylinux_i686 cp39-manylinux_i686 pp* *-win32 cp310-manylinux_i686"
```

Not sure this wildcard will work, but:

```suggestion
        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* pp* *-win32 *-manylinux_i686"
```